projects/driver_controls/inc/x86/ads1015.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+// Module for using ads1015
+// GPIO and Interrupt, I2C should be initiated
+// The user also needs to create a struct ADS1015Data which would persist accross functions
+// start the ads1015 using ads1015_init and then configure channels using ads1015_config_channel
```

Comment:
I think these sentences should be capitalized? I thought they were from the sentence above at first glance.